src/com/company/Board.java

```diff
@@ -0,0 +1,100 @@
+package com.company;
+
+import java.util.Arrays;
+
+public class Board implements IBoard {
+
+  private final int Row;
```

Comment:
property name should be lowercase. You already know that so why starting with capital letter?

Reply:
ok, will fix that